dev/WindowsAppSDK_DLL/WindowsAppSDK_DLL.vcxproj

```diff
@@ -394,6 +395,9 @@
   <ImportGroup Label="ExtensionTargets">
     <Import Project="..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.210204.1\build\native\Microsoft.Windows.ImplementationLibrary.targets" Condition="Exists('..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.210204.1\build\native\Microsoft.Windows.ImplementationLibrary.targets')" />
     <Import Project="..\..\packages\Microsoft.Windows.CppWinRT.2.0.210403.2\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('..\..\packages\Microsoft.Windows.CppWinRT.2.0.210403.2\build\native\Microsoft.Windows.CppWinRT.targets')" />
+    <Import Project="..\..\packages\Microsoft.FrameworkUdk.amd64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.amd64fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.amd64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.amd64fre.targets')" />
+    <Import Project="..\..\packages\Microsoft.FrameworkUdk.arm64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.arm64fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.arm64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.arm64fre.targets')" />
+    <Import Project="..\..\packages\Microsoft.FrameworkUdk.x86fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.x86fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.x86fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.x86fre.targets')" />
```

Comment:
Is this how we use the FrameworkUdk - N references (1 for each architecture)? Asking for a friend :P

Reply:
This is how it is published (1 for each arch). Could be better but that's on Ben's team.
Also how MRTCore uses the FrameworkUDK.